tests/ssl-opt.sh

```diff
@@ -5535,6 +5535,13 @@ run_test    "DTLS fragmenting: gnutls server, DTLS 1.0" \
             -c "fragmenting handshake message" \
             -C "error"
 
+# We use --insecure for the GnuTLS client because it expects
```

Comment:
Isn't this considered to be a temporary workaround? Sorry for not following the discussion on this.

Reply:
I expect it to be temporary - once GnuTLS allows specifying the server name expected in the certificate separately, we don't need to resort to `--insecure` here.